tests/rptest/wasm/native_kafka_producer.py

```diff
@@ -33,6 +36,9 @@ def task_name(self):
         return f"producer-worker-{str(random.randint(0, 9999))}"
 
     def _run(self):
+        producer = KafkaCliTools(self._redpanda)
+        producer.produce(self._topic, self._num_records, self._records_size)
```

Comment:
Technically this class shouldn't be called `NativeKafkaProducer` if your going to use `kafkaCliTools` as the implementation. `native` means it uses the `kafka-python` official API under the hood